_rules/word-spacing-not-important-9e45ec.md

```diff
@@ -0,0 +1,160 @@
+---
+id: 9e45ec
+name: Word spacing in `style` attributes is not `!important`
+rule_type: atomic
+description: |
+	This rule checks that the `style` attribute is not used to prevent adjusting `word-spacing` spacing by using `!important`, except if it's at least `0.16` times the font size.
```

Comment:
```suggestion
	This rule checks that the `style` attribute is not used to prevent adjusting `word-spacing` by using `!important`, except if it's at least `0.16` times the font size.
```